### ISSUE TITLE
Plugin is broken due to API change

### ISSUE THREAD

**Issue by tareko** [tareko/discourse-replyto-individual#1]
As of ~ August, this plugin broke due to a change in the API> The error message in sidekiq is:

```
Jobs::HandledExceptionWrapper: Wrapped NameError: undefined local variable or method `reply_key' for #<Email::MessageBuilder:0x00007f51bf7257f0>
```

**Comment by tareko** [comment]
Fixed now

**Comment by tareko** [comment]
Fixed fully by 9e40992149def92f4cfc659f9706c6d20d10a22d
